src/docs/build-windows.dox

```diff
@@ -16,27 +16,34 @@ Now proceed with @ref windows_building
 @section windows_building Building on Windows
 
 Building WiredTiger on Windows requires
-<a href="http://www.scons.org">SCons</a> as well as the Microsoft Visual
+<a href="https://cmake.org/">CMake</a> as well as the Microsoft Visual
 C++ compiler in Microsoft Visual Studio.
 
-Change directory to the top-level directory, then build the software:
+You can build WiredTiger from source using command-line tools.
+When compiling via command-line tools, we recommend a prompt/shell that has been appropriately
+configured with VS toolchain environment variables. We usually recommend using the
+<a href="https://docs.microsoft.com/en-us/visualstudio/ide/reference/command-prompt-powershell">Visual Studio Developer Command Prompt/Powershell</a> to
+ensure an appropriately configured environment.
+
+Change directory to the top-level directory, then create and configure a new directory to run your
+build from:
 
 @code
 cd wiredtiger
-scons
+mkdir build
+cd build
 @endcode
 
-To rebuild from scratch, discard any previous configuration by cleaning
-out the build area:
+Change into your newly created build directory and using CMake run the build configuration
+step to generate your build.
 
 @code
-scons -c
+cmake ..\.
 @endcode
 
-To see additional configuration options, run:
-
+Lastly, in the same directory you configured your build, start the build:
 @code
-scons --help
+msbuild ALL_BUILD.vcxproj
```

Comment:
I assume `ALL_BUILD.vcxproj` is generated by cmake?  I don't see that file referred to or created directly by anything in this branch.

Reply:
Yep, by default, CMake generates `vcxproj` files to invoke the build. 
Sorry for the confusion, to make this more explicit I expanded on the origin of the Visual Studio project files in the doc ac59ae5dd9a1589869f446fee98fa19105a69f2d